src/components/particules/SizeInstance.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div
+    class="select-cell rounded-lg mr-2 border d-flex justify-center align-center"
+    :style="{ backgroundColor: '#F3F3F3' }"
+  >
+    <p class="text-caption font-weight-bold">{{ size }}</p>
+  </div>
+</template>
+
+<script setup>
+// eslint-disable-next-line no-unused-vars
+const props = defineProps({
+  size: String,
+  isAvailable: Boolean,
```

Comment:
@balanejacob We can call this just `available`, to follow the same pattern as the HTML elements where they accept `disabled` or `checked`.